### PR TITLE
Enable (even force :p) TLS everywhere !

### PR DESCRIPTION
**Pull request by elafarge**
No worries though, everything is automated thanks to Let's Encrypt !

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,6 @@ Session.vim
 *~
 # auto-generated tag files
 tags
+
+# Heheheheh
```
Comment (hkaj) [inline .gitignore:14]: top notch comment right there 👌

File: traefik.toml [hunk 2]
```diff
@@ -28,20 +28,20 @@ debug = true
...
 #
-# [acme]
+[acme]
 
 # Email address used for registration
 #
 # Required
 #
-# email = "foo@bar.com" # TODO: set that as a CLI argument
+email = "nick@tam.er" # Just set that as a CLI argument!
```
Comment (hkaj) [inline traefik.toml:44]: :man_facepalming:

File: docker-compose.yaml [hunk 3]
```diff
@@ -3,15 +3,17 @@ version: '3'
...
     image: traefik
-    command: --web --web.address=":${ADMIN_PORT}" --entrypoints="Name:http Address::${HTTP_ENTRYPOINT_PORT}" --docker --docker.domain="${DOCKER_DOMAIN}" --logLevel=DEBUG
+    command: --web --web.address=":${ADMIN_PORT}" --docker --docker.domain="${DOCKER_DOMAIN}" --docker.usebindportip=true --docker.exposedbydefault=false --acme.email="${ACME_EMAIL}"
     networks:
-      - default
+    - default
     ports:
-      - "${HTTP_ENTRYPOINT_PORT}:${HTTP_ENTRYPOINT_PORT}"
-      - "${ADMIN_PORT}:${ADMIN_PORT}"
+    # - "80:80" # Uncomment this to enable automatic HTTP-->HTTPs redirection
```
Comment (hkaj) [inline docker-compose.yaml]: we should uncomment that since we have redirection enabled in traefik.toml

File: .gitignore (hunk 1 above)
Comment (elafarge) [reply .gitignore:14]: :$

File: traefik.toml (hunk 2 above)
Comment (elafarge) [reply traefik.toml:44]: :disappointed:

File: docker-compose.yaml (hunk 3 above)
Comment (elafarge) [reply docker-compose.yaml]: Yep, I just figured out today it was a shitty idea. That's updated :)

**Comment by elafarge** [comment]
Rebased and force-pushed @hkaj

**Pull request merged**